integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/deploy/Deploys.java

```diff
@@ -187,6 +194,23 @@ private void initBaseProducer() {
     baseProducer = new MicroserviceDeploy(definition);
   }
 
+
+  private void initBaseHttp2Producer() {
+    MicroserviceDeployDefinition definition = new MicroserviceDeployDefinition();
+    definition.setDeployName("baseHttp2Producer");
+    definition.setCmd("it-producer");
+    definition.setArgs(new String[] {});
+    definition.setArgs(new String[] {"-Dservicecomb.rest.address=0.0.0.0:0?protocol=http2",
+        "-Dservicecomb.highway.address=0.0.0.0:0?protocol=http2"});
```

Comment:
do need to overide highway address

only test h2c, not test h2?

Reply:
do not need ? or do need ?

Reply:
you means do not need to overide ?

Reply:
you means do not need to overide ?